contracts/src/v0.8/functions/tests/1_0_0/FunctionsCoordinator.t.sol

```diff
@@ -0,0 +1,219 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+// ================================================================
+// |                       Functions Coordinator                  |
+// ================================================================
+
+/// @notice #constructor
+contract FunctionsCoordinator_Constructor {
+
+}
+
+/// @notice #getThresholdPublicKey
+contract FunctionsCoordinator_GetThresholdPublicKey {
+
+}
+
+/// @notice #setThresholdPublicKey
+contract FunctionsCoordinator_SetThresholdPublicKey {
+
+}
+
+/// @notice #getDONPublicKey
+contract FunctionsCoordinator_GetDONPublicKey {
+
+}
+
+/// @notice #setDONPublicKey
+contract FunctionsCoordinator__SetDONPublicKey {
+
+}
+
+/// @notice #_isTransmitter
+contract FunctionsCoordinator_IsTransmitter {
+
+}
+
+/// @notice #setNodePublicKey
+contract FunctionsCoordinator_SetNodePublicKey {
```

Comment:
Shouldn't all those contracts inherit from a base one? Either Test.sol or some Functions-specific one?
I'm not convinced that having those empty stubs is the best way to init test migration. Maybe just list all methods in a comment?